android/titanium/src/java/org/appcelerator/titanium/TiC.java

```diff
@@ -1447,6 +1447,11 @@
 	/**
 	 * @module.api
 	 */
+	public static final String PROPERTY_FREE_SPACE = "freeSpace";
```

Comment:
Good idea! I'll go for `hiddenBehaviour: Ti.UI.VIEW_GONE | Ti.UI.VIEW_INVISIBLE`
Then it looks almost like the native View.GONE and View.INVISIBLE